projects/plugins/jetpack/_inc/client/security/waf.jsx

```diff
@@ -498,6 +537,10 @@ export default connect(
 		return {
 			automaticRulesAvailable: getAutomaticRulesAvailable( state ),
 			hasScan: siteHasFeature( state, 'scan' ),
+			protectAdminUrl: `${ getSiteAdminUrl( state ) }admin.php?page=jetpack-protect#/firewall`,
+			isProtectInstalled: PROTECT_PLUGIN_FILES.some( pluginFile =>
+				isPluginInstalled( state, pluginFile )
```

Comment:
Looks like there is an action to check for an active state that might be more applicable here:
```suggestion
				isPluginActive( state, pluginFile )
```

Reply:
Good catch! Applied via https://github.com/Automattic/jetpack/pull/38655/commits/c63adbc9cc0716cb78722c70b5c4025d219fe7da :+1: